crates/events/README.md

```diff
@@ -20,6 +20,6 @@
   <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
 </div>
 
-Using event listeners with [`web-sys`](https://crates.io/crates/web-sys) is hard! This crate provides an [`EventListener`](https://docs.rs/gloo-events/0.1.0/gloo_events/struct.EventListener.html) type which makes it easy!
+Using event listeners with [`web-sys`](https://crates.io/crates/web-sys) is hard! This crate provides an [`EventListener`](struct.EventListener.html) type which makes it easy!
```

Comment:
These changes seem wrong, since it will cause the URL to point to `https://crates.io/crates/gloo-events/struct.EventListener.html`, not `https://docs.rs/gloo-events/0.1.1/gloo_events/struct.EventListener.html` like it should.

However, I would be in favor of changing the URL to use `^0.1.0` instead, so it will point to the latest version.

Reply:
Hmm, I don't remember making this change. Are the READMEs auto-generated? Anyway it's no trouble to revert it. if necessary.

Reply:
I've changed it to `^0.1.0` like you suggested.